<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuColumn">
     <property name="title">
      <string>&amp;Column</string>
     </property>
     <addaction name="actionAdd_One"/>
    </widget>
    <widget class="QMenu" name="menuRow">
     <property name="title">
      <string>&amp;Row</string>
     </property>
     <addaction name="actionAdd_One_2"/>
    </widget>
    <addaction name="menuColumn"/>
    <addaction name="menuRow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_as">
   <property name="text">
    <string>&amp;Save As ...</string>
   </property>
  </action>
  <action name="actionAdd_One">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="actionAdd_One_2">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
